ArduCopter/GCS_Mavlink.cpp

```diff
@@ -1377,7 +1377,7 @@ void GCS_MAVLINK_Copter::handleMessage(mavlink_message_t* msg)
         mavlink_msg_set_position_target_local_ned_decode(msg, &packet);
 
         // exit if vehicle is not in Guided mode or Auto-Guided mode
-        if ((copter.control_mode != GUIDED) && !(copter.control_mode == AUTO && copter.flightmode_auto.mode() == Auto_NavGuided)) {
```

Comment:
`flightmode->accepts_set_destination_and_set_velocity()`?

Reply:
yup, let's do in follow up PR.